packages/instant/src/redux/reducer.ts

```diff
@@ -180,13 +182,14 @@ export const createReducer = (initialState: State) => {
                 }
                 return state;
             case ActionTypes.SetBuyOrderStateSuccess:
-                const successTxHash = action.data;
+                const successTxHash = action.data.txHash;
```

Comment:
Was this not correct??

Reply:
it was, but I refatcored `data` to be an object now